backend/debug_runner.py

```
@@ -0,0 +1,76 @@
+
+    # Add job to store first so update_job doesn't fail on "not found"
+    from store import add_job
+    add_job(job)
+
+    runner = SandboxKilocodeRunner()
+    
+    print("Invoking runner.start()...")
+    try:
+        await runner.start(job, plan, cluster)
+        print("Runner finished successfully (check status in logs)")
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)